music21/spanner.py

```diff
@@ -471,6 +471,46 @@ def addSpannedElements(
 
         self.spannerStorage.coreElementsChanged()
 
+    def insertFirstSpannedElement(self, firstEl: base.Music21Object):
+        '''
+        Add a single element as the first in the spanner.
+
+        >>> n1 = note.Note('g')
+        >>> n2 = note.Note('f#')
+        >>> n3 = note.Note('e')
+        >>> n4 = note.Note('d-')
+        >>> n5 = note.Note('c')
+
+        >>> sl = spanner.Spanner()
+        >>> sl.addSpannedElements(n2, n3)
+        >>> sl.addSpannedElements([n4, n5])
+        >>> sl.insertFirstSpannedElement(n1)
+        >>> sl.getSpannedElementIds() == [id(n) for n in [n1, n2, n3, n4, n5]]
+        True
+        '''
+        from music21 import stream
```

Comment:
stream is only used for typing, so please put at the top of the module:

```
if t.TYPE_CHECKING:
    from music21 import stream
```

There is an overhead to these in-function imports.

Reply:
ok

Reply:
Done.